src/styles/views/Game.scss

```diff
@@ -9,51 +9,57 @@
 
 .game {
   &.container {
-    background: black;
-    padding: 1.5em;
+    margin: 1em;
     border-radius: $borderRadius;
     display: flex;
     flex-direction: column;
     align-items: center;
+    background-color: $background;
     box-shadow: $dropShadow;
   }
-  &.user-list {
-    list-style: none;
-    padding-left: 0;
-  }
-  &.user-item {
+}
+
+.wordmergebar {
+  &.container {
+    border-radius: $borderRadius;
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
     align-items: center;
-    justify-content: center;
+    background-color: #454545;
+    margin: 0.5em;
+    box-shadow: $dropShadow;
   }
-  & p,h2 {
-    margin: 0.5em 0;
+
+  &.symbol {
+    font-size: 5em;
+    text-align: center;
+    display: flex;
   }
 }
 
-.player {
+.wordboard {
   &.container {
-    margin: 0.5em 0;
-    width: 20em;
-    padding: 10px;
+    padding: 0.5em;
     border-radius: $borderRadius;
     display: flex;
+    flex-direction: column;
     align-items: center;
-    background: lighten($background, 5);
-  }
-  &.name {
-    font-weight: 300;
-    margin-left: 5px;
-  }
-  &.username {
-    font-weight: 500;
-    color: $accent;
-  }
-  &.id {
-    margin-left: auto;
-    margin-right: 10px;
-    font-family: $fontMono;
-    font-weight: 300;
+    background-color: #101010;
+    margin: 0.5em;
+    box-shadow: $dropShadow;
   }
+}
+
+.wordrow {
+  display: flex;
+  flex-direction: row;
+}
+
+.word {
+  padding: 0.5em;
+  border-radius: $borderRadius/2;
+  display: flex;
+  align-items: center;
+  margin: 0.5em;
+  box-shadow: $dropShadow;
 }
```

Comment:
To avoid having these 'empty buttons' that are buttons with no content, you can either display none of them or display them and make them invisible. So just add this code:

```js
.word:empty {
  opacity: 0;  //or display:none; So if the primary-button of class word has no string in it, it'll do this.
}
```

Reply:
Thanks for the suggestion! Now that I think about it, maybe it would make more sense to show the buttons, but have them with some spaces or placeholder characters? Like this:

![image](https://github.com/sopra-fs24-group-41/sopra-fs24-group-41-client/assets/93321296/8cbf3539-f5b4-4b88-a09b-97e8ed75022d)

This way, it would be clearer for the user that they need to add a second word, and that something comes out at the end.